.env

```diff
@@ -0,0 +1,28 @@
+# In all environments, the following files are loaded if they exist,
```

Comment:
should this file be committed?

Reply:
Allright, that is one of the things I need to address. Will fix this in a new PR, okay?